### PR TITLE
Badged up the README

### PR DESCRIPTION
**Pull request by Mingling94**
Awww yeah

selection_034

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,11 @@
-# Sia UI - A Highly Efficient Decentralized Storage Network
+# [![Sia Logo](http://sia.tech/resources/img/svg/sia-green-logo.svg)](http://sia.tech/)-UI
```
Comment (DavidVorick) [inline README.md]: The `-UI` looks silly

Comment (Mingling94) [inline README.md]: It totally is silly

Comment (DavidVorick) [inline README.md]: let us do something less silly

Comment (Mingling94) [inline README.md]: It's less silly :P

**Pull request merged**